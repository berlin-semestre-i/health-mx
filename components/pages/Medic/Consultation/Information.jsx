import React from 'react'
import { Form } from 'semantic-ui-react'
import Card from '../../../shared/Card'
import styled from 'styled-components'

const Information = ({ motive, physicalExploration, conclusion, onChange }) => (
  <React.Fragment>
    <Card header="Información de Consulta" className="nth-col-first">
      <Form name="information">
        <FormTextArea
          name="motive"
          label="Motivo de la consulta:"
          placeholder="Escribe aquí el motivo de la consulta."
          value={motive}
          onChange={onChange}
        />
        <FormTextArea
          name="physicalExploration"
          label="Exploración física:"
          placeholder="En caso de aplicar,
                    ingresa las observaciones de la exploración física."
          value={physicalExploration}
          onChange={onChange}
        />
        <FormTextArea
          name="conclusion"
          label="Observaciones y conclusiones de la consulta:"
          placeholder="Escribe aquí las observaciones finales de la consulta."
          value={conclusion}
          onChange={onChange}
        />
      </Form>
    </Card>
  </React.Fragment>
)

export default Information

const FormTextArea = styled(Form.TextArea)`
   & textarea {
    font-family: -apple-system,system-ui,BlinkMacSystemFont, 
    "Segoe UI",Roboto,'Helvetica Neue',Arial,Helvetica,sans-serif;
   }
`