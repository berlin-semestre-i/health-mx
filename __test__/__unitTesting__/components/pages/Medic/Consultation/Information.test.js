import React from 'react'
import { shallow } from 'enzyme'
import Information from '../../../../../../components/pages/Medic/Consultation/Information'

describe('Information', () => {
  it('renders', () => {
    const component = shallow(<Information />)
    expect(component).toMatchSnapshot()
  })
})