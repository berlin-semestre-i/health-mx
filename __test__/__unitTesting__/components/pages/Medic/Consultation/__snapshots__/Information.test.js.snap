// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Information renders 1`] = `
<Fragment>
  <CustomCard
    className="nth-col-first"
    header="Información de Consulta"
  >
    <Form
      as="form"
      name="information"
    >
      <Information__FormTextArea
        label="Motivo de la consulta:"
        name="motive"
        placeholder="Escribe aquí el motivo de la consulta."
      />
      <Information__FormTextArea
        label="Exploración física:"
        name="physicalExploration"
        placeholder="En caso de aplicar, ingresa las observaciones de la exploración física."
      />
      <Information__FormTextArea
        label="Observaciones y conclusiones de la consulta:"
        name="conclusion"
        placeholder="Escribe aquí las observaciones finales de la consulta."
      />
    </Form>
  </CustomCard>
</Fragment>
`;
